### DISCUSSION TITLE
Disabling mobile styles on hover?

### DISCUSSION THREAD

**Discussion by rightaway** [tailwindlabs/tailwindcss#1560] [General]
If we use javascript to detect that a device is a mobile device and set a CSS class on the `<html>` tag, could tailwind somehow make it possible to disable its `hover` styles for that device? There's several options in https://stackoverflow.com/questions/23885255/how-to-remove-ignore-hover-css-style-on-touch-devices but I would like to see if anyone has ideas of whether tailwind can help with such a use case.

**Comment by cossssmin** [comment]
A utility-first approach might be registering new screens:

```js
module.exports = {
  // ...
  screens: {
    touch: {'raw': 'hover: none'},
    'can-hover': {'raw': 'hover: hover'}
  }
}
```

So you could write `can-hover:underline` instead of the default `hover:underline` for devices that support hover, or `touch:no-underline` if you want to keep using `hover:` and then negate it.

**Reply by csenio** [reply]
@cossssmin you still need to write `can-hover:hover:underline` right?

**Comment by ghost** [comment]
I solved this problem by using https://stackoverflow.com/a/67458852/16381640
